1_basics.ipynb

```diff
@@ -25,20 +43,57 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# What is HSR?\n",
+    "\n",
+    "HSR (HSR: Human Support Robot) is one of the \"Toyota Partner Robots\" that support human activities, and it will assist independent living at home for people with disabilities.\n",
+    "\n",
+    "In this notebook, you will learn the overview of the robot by using the HSR simulator."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "# Jupyter Notebookの使い方"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# How to use Jupyter Notebook"
```

Comment:
If referring to a single previously unspecified Jupyter Notebook: 
"# How to use **a** Jupyter Notebook"

If referring to more than one Jupyter Notebook in a general sense: 
"# How to use Jupyter Notebooks"

If referring to more than one previously referred to Jupyter Notebooks: 
"# How to use **the** Jupyter Notebooks"

Reply:
I think this part will describe Jupyter Notebook in general.

The term "Jupyter Notebook" is the name for the software (not the individual notebooks). Do we need to add "s" in this case?

Reply:
Ah, yes.  In this case the following looks good:

"# How to use Jupyter Notebooks"